### PR TITLE
Website update

### PR DESCRIPTION
**Pull request by RoseZhao**
This pull request updates the website.  For a full description, please refer to https://github.com/scikit-image/scikit-image/pull/2585.

1. Vertically adjust search documentation bar to be in the middle.
2. Take away Google Plus +1 and expand the GitHub Star to fill its place.
3. Add Contribute under "Links" on home page. Point it to the Contributor page in documentation.
4. Move Announcement above "Getting Started" and rename it to "News".
5. Move all the sidebars on the right to the left on every page.
6. Remove the "home" tab at the home page (click on skimage icon instead).
7. Reorder the blocks on home page: introduction, citation, news, getting started.
8. Add a sentence at the end of getting started: "read more in our user guide". Make a hyperlink to the user guide page in documentation.
9. Remove "Related Projects".
10. Fix the horizontal alignment issue with the top and second block (off by one or two pixels).
11. Remove easy_install (only leave pip) and add installation instructions for Conda on Download page.
12. Put "Stable" and "Development" in one block.

### CODE REVIEW

**Comment by soupault** [comment]
@RoseZhao Could you provide a description for this PR?

**Comment by stefanv** [comment]
@soupault @RoseZhao has been working with me on improving the website.  I think something went wrong while setting up this PR, since it doesn't contain even nearly all the updates she has made.  I'll help her update this branch as soon as she is back (it's spring break at Berkeley right now).

**Comment by RoseZhao** [comment]
Scikit-Image Web Update

File: _templates/sidebar_versions.html [hunk 1]
```diff
@@ -21,28 +21,25 @@
 </script>
 
 <div class="well">
+    <strong>Current Version: Skimage 0.12<br></strong>
```
Comment (stefanv) [inline _templates/sidebar_versions.html]: Remove this box, add this exact thing to the documentation and remove the doc versions there.

File: index.rst [hunk 2]
```diff
@@ -76,7 +49,8 @@ visit our `gallery </docs/dev/auto_examples>`__.
        http://dx.doi.org/10.7717/peerj.453
        </a>
 
-Announcements
+
+News
 -------------
```
Comment (stefanv) [inline index.rst]: Update this to four dashes to match length of News

File: themes/scikit-image/static/css/bootstrap.min.css [hunk 3]
```diff
@@ -6,4 +6,4 @@
...
...-tabs .active>a,.tabs-right>.nav-tabs .active>a:hover{border-color:#ddd #ddd #ddd transparent;*border-left-color:#fff}.nav>.disabled>a{color:#999}.nav>.disabled>a:hover{text-decoration:none;cursor:default;background-color:transparent}.navbar{*position:relative;*z-index:2;margin-bottom:20px;overflow:visible;color:#777}.navbar-inner{min-height:40px;padding-right:20px;padding-left:20px;background-color:#fafafa;background-image:-moz-linear-gradient(top,#fff,#f2f2f2);background-image:-webkit-gradient(linear,0 0,0 100%,from(#fff),to(#f2f2f2));background-image:-webkit-linear-gradient(top,#fff,#f2f2f2);background-image:-o-linear-gradient(top,#fff,#f2f2f2);background-image:linear-gradient(to bottom,#fff,#f2f2f2);background-repeat:repeat-x;border:1px solid #d4d4d4;-webkit-border-radius:4px;-moz-border-radius:4px;border-radius:4px;filter:progid:dximagetransform.microsoft.gradient(startColorstr='#ffffffff',endColorstr='#fff2f2f2',GradientType=0);*zoom:1;-webkit-box-shadow:0 1px 4px rgba(0,0,0,0.065);-moz-box-shadow:0 1px 4px rgba(0,0,0,0.065);box-shadow:0 1px 4px rgba(0,0,0,0.065)}.navbar-inner:before,.navbar-inner:after{display:table;line-height:0;content:""}.navbar-inner:after{clear:both}.na
```
Comment (stefanv) [inline themes/scikit-image/static/css/bootstrap.min.css]: Double check why this change was made / needed.  Ideally, we don't want to modify bootstrap at all.
